packages/@okta/vuepress-site/use_cases/inline_hooks/saml_hook/saml_hook/index.md

```diff
@@ -0,0 +1,405 @@
+          },
+          "factors": {
+            "href": "https://{yourOktaDomain}/api/v1/users/00uq8tMo3zV0OfJON0g3/factors"
+          }
+        }
+      }
+    },
+    "assertion": {
+      "subject": {
+        "nameId": "administrator1@clouditude.net",
+        "nameFormat": "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified",
+        "confirmation": {
+          "method": "urn:oasis:names:tc:SAML:2.0:cm:bearer",
+          "data": {
+            "recipient": "http://www.example.com:7070/saml/sso"
+          }
+        }
+      },
+      "authentication": {
+        "sessionIndex": "id1553800523546.312669168",
+        "authnContext": {
+          "authnContextClassRef": "urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport"
+        }
+      },
+      "conditions": {
+        "audienceRestriction": [
+          "urn:example:sp"
+        ]
+      },
+      "claims": {
+        "foobie": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "doobie"
+            }
+          ]
+        },
+        "array": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "Array 1"
+            },
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "Array2"
+            },
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "Array3"
+            }
+          ]
+        },
+        "middle": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "middellllll"
+            }
+          ]
+        },
+        "firstAndLast": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "7d6a50c8-4d7e-4058-9c5b-2cc98cecd294"
+            }
+          ]
+        }
+      },
+      "lifetime": {
+        "expiration": 300
+      }
+    }
+  },
+  "eventTypeVersion": "1.0",
+  "cloudEventVersion": "0.1",
+  "eventType": "com.okta.saml.tokens.transform",
+  "contentType": "application/json"
+}
+```
+
+## Sample Listing of JSON Payload of Response
+
+```json
+{
+  "error": null,
```

Comment:
It is better to remove the object with `null` value.

Reply:
Removed.